src/Payment.ts

```diff
@@ -157,27 +160,37 @@ export class Payment {
     value: number | string,
     options: PaymentOptions = {}
   ): Promise<PathObject> {
-    const { networkDecimals, maximumHops, maximumFees } = options
+    const {
+      networkDecimals,
+      maximumHops,
+      maximumFees,
+      feePayer: feePayerOption
+    } = options
     const decimals = await this.currencyNetwork.getDecimals(networkAddress, {
       networkDecimals
     })
     const data = {
+      feePayer: feePayerOption,
       from: senderAddress,
       maxFees: maximumFees,
       maxHops: maximumHops,
       to: receiverAddress,
       value: utils.calcRaw(value, decimals.networkDecimals).toString()
     }
     const endpoint = `networks/${networkAddress}/path-info`
-    const { estimatedGas, fees, path } = await this.provider.fetchEndpoint<
-      PathRaw
-    >(endpoint, {
+    const {
+      estimatedGas,
+      fees,
+      path,
+      feePayer
+    } = await this.provider.fetchEndpoint<PathRaw>(endpoint, {
       body: JSON.stringify(data),
       headers: new Headers({ 'Content-Type': 'application/json' }),
       method: 'POST'
     })
     return {
       estimatedGas: new BigNumber(estimatedGas),
+      feePayer: feePayer as FeePayer,
```

Comment:
I think we should check this to make sure this is true. Otherwise types will not help us. 
It should be possible with something like:
`if Object.values(FeePayer).includes(feePayer)`